test-data/unit/fine-grained-blockers.test

```diff
@@ -407,3 +407,49 @@ a.py:1: error: invalid syntax
 ==
 b.py:2: error: Incompatible return value type (got "str", expected "int")
 ==
+
+[case testDecodeErrorBlocker-posix]
+import a
+a.f(1)
+[file a.py]
+def f(x: int) -> None: ...
+[file a.py.2]
+# coding: ascii
+ä
```

Comment:
I'd make this an assignment (which would be legal with the Py3 default encoding).